examples/pruning/torch/resnet18/README.md

```diff
@@ -32,4 +32,6 @@ It's pretty simple. The example does not require additional preparation. It will
 
 ```bash
 python main.py
+# Or to run Regularization-Based pruning
+python main.py --mode rb
```

Comment:
the [documentation about sparsity](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/other_algorithms/Sparsity.md
) is not connected with this sample and contains old configs.
are you going to update in the next PRs?